File: TSPractice/P8.ts
//Program to Find if a Given Year is a Leap Year

let year = 2000;
let leapYear = false;
if(year % 4 === 0){
    if(year % 100 === 0){
        if(year % 400 === 0){
            leapYear = true;
        } else {
            leapYear = false;
        }
    } else {
        leapYear = true;
    }
}

if(leapYear) {
    console.log("Giver year is a leap year!")
} else {
    console.log("Giver year is not a leap year!")
}

File: Java/src/BasicProblems/P8.java
//Program to Find LCM of Two Numbers

public class P8 {
    public static void main(String[] args) {
        int a = 15;
        int b = 25;
        int largest = a > b ? a : b;
        while( true ) {
            if (largest % a == 0 && largest % b ==0 )
                break;
            largest++;
        }
        System.err.println("LCM of given number is "+ largest);
    }
}
